01.fit_a_line/README.md

```diff
@@ -212,14 +210,9 @@ def event_handler(event):
                 print('loss is less than 10.0, stop')
                 trainer.stop()
 
-            if step >= 2000:
```

Comment:
Removed. It takes a few minutes to converge. Only a bit longer than `step == 2000`

![screen shot 2018-05-30 at 8 28 05 pm](https://user-images.githubusercontent.com/5635322/40759831-a2321e1c-6448-11e8-92bc-aadbfce6c779.png)